src/client/driver/index.js

```diff
@@ -10,6 +10,7 @@ import * as browser from '../browser';
 import executeActionCommand from './command-executors/execute-action-command';
 import executeWaitForElementCommand from './command-executors/execute-wait-for-element-command';
 import executeNavigateToCommand from './command-executors/execute-navigate-to-command';
+import executePendingBrowserWindowCommand from './command-executors/execute-pending-browser-window-command';
```

Comment:
TestRun now has `pendingWindowManipulation `.
Which command we should have `RunWindowManipulationCommand` or `PendingWindowManipulationCommand`?
Function on client `executePendingWindowManipulationCommand` or `executeRunWindowManipulationCommand`?